### ISSUE TITLE
client axios -> graphQL query -> prisma -> psql associative data not getting added to  [10:40pm]

### ISSUE THREAD

**Issue by frankcollins3** [frankcollins3/mine-nugget-ts#160]
attempting to do:
titled.

**error:**
👍 INSERT INTO "MinersOnStrains' VALUES (1, 1) 
![Screen Shot 2023-08-04 at 10 38 27 PM](https://github.com/frankcollins3/mine-nugget-ts/assets/73137934/b521319a-7646-4487-8a12-009a79f5504d)
![Screen Shot 2023-08-04 at 10 39 42 PM](https://github.com/frankcollins3/mine-nugget-ts/assets/73137934/d63dd58f-da8e-458b-a171-0a796cf54ae4)
![Screen Shot 2023-08-04 at 10 38 20 PM](https://github.com/frankcollins3/mine-nugget-ts/assets/73137934/966e7d09-7195-4484-ab51-15c2b5a5fd5a)
![Screen Shot 2023-08-04 at 10 38 14 PM](https://github.com/frankcollins3/mine-nugget-ts/assets/73137934/5c525333-9661-40d8-8d48-95bcc6c1c051)

👍  { minersId: 0, strainsid: 0 } get added to the DB so it's not a 400 or 500 graphQL error

**proposed approach:**
// ended up going with strainsid instead of strainsId because of trouble seeding the database with capitalizations.
fixing the seeding with such situation seems easier than this.

1: clear tables and reconstruct psql columns and prisma models

**Comment by frankcollins3** [comment]
this also means changing the function that redeclares object endpoints in Promises.tsx?
the func that lets the user iterate through the strains API during SeeAndSave

**Comment by frankcollins3** [comment]
message
: 
"\nInvalid `prisma.minersOnStrains.create()` invocation:\n\n{\n  where: {\n  ~~~~~\n    data: {\n      minersId: 1,\n      strainsid: 1\n    }\n  },\n+ data: MinersOnStrainsCreateInput | MinersOnStrainsUncheckedCreateInput\n}\n\nUnknown arg `where` in where for type MinersOnStrains. Did you mean `data`?\nArgument data is missing.\n\nNote: Lines with + are

**Comment by frankcollins3** [comment]
tail end of day yesterday prisma doesn't have a where statement.
all I saw were the words: "youre right on track!" and tried for 60 mins to get it working. 

woke up, planned to dismantle prisma tables and reset, asked AI about relations and whether it was targeting:
strain.id or strain.strainid

removed the .catch() block because it returns an object error free.
looked at the error. No "WHERE" statement in the create invocation.

![Screen Shot 2023-08-05 at 7 48 28 AM](https://github.com/frankcollins3/mine-nugget-ts/assets/73137934/58c8cbda-7ca3-433c-bdbc-33b2bd9e4af4)

![Screen Shot 2023-08-05 at 7 48 36 AM](https://github.com/frankcollins3/mine-nugget-ts/assets/73137934/57c07202-12b8-4207-a37e-633a90c38ace)

// AI did correct me & all I looked at was: "You're on the right track!" having done bunch of .create()... slipped and fell
[7:51am]
